apps/angular/project.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "devextreme-angular-playground",
```

Comment:
Let's delete `project.json` file for all playgrounds
I think default inputs and outputs will look more simple and more stable

Reply:
Inputs - may be. But outputs - no.